frontend/src/app/libs/event-description/models/event-description.ts

```diff
@@ -0,0 +1,6 @@
+import { Event } from '../../landing/models/event';
+
+export interface EventDescription {
```

Comment:
user login status must be obtained from auth facade and placed in store, not need to create interface that fill averything.